### ISSUE TITLE
Modifiable property used in_array() should not be inlined

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#1614]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | U3.0.10 |
| Language level | PHP 7.4                 |

Current behaviour
--

```php
class TestCase
{
    public $array = [ '123' ];

    public function is456(): bool
    {
        // Warning: [EA] '456' == '123' should be used instead.
        return in_array('456', $this->array);
    }
}
```

Expected behaviour
--

No warning.

Details
--

Basically, it suggests to inline a property that could be modified in runtime, which will changes the code behavior.

**Comment by kalessil** [comment]
Fixed!

**Comment by rentalhost** [comment]
@kalessil is this fix already available? It still happens to me.
